python/pyspark/sql/functions.py

```diff
@@ -91,13 +92,58 @@ def lit(col):
     Creates a :class:`Column` of literal value.
 
     .. versionadded:: 1.3.0
+    .. versionchanged:: 3.2.0
+        Added support for complex type literals.
+
+    Parameters
+    ----------
+    col : bool, float, int, str, datetime.date, datetime.datetime, dict, list, tuple
+        Object to be converted into :class:`Column`.
```

Comment:
"Creates a :class:`Column` of literal value."?

https://github.com/apache/spark/blob/72b7f8abfb60d0008f1f9bed94ce1c367a7d7cce/sql/core/src/main/scala/org/apache/spark/sql/functions.scala#L105

Reply:
I am not sure if I understand what you mean. We already have matching sentence in the description block

https://github.com/apache/spark/blob/6198e0aca0d30d6bd6ba9f29a0fe60a62016caed/python/pyspark/sql/functions.py#L92

Could you clarify please?

Reply:
Ah, nvm. I missed it.